subprojects/emcee/worker/src/main/kotlin/com/avito/emcee/worker/StartWorkerCommand.kt

```diff
@@ -68,7 +69,11 @@ internal class StartWorkerCommand(
             .addHandler(ProcessingBucketsRequestHandler(bucketsStorage))
             .debug(debugMode)
             .build()
-            .start(config.workerPort)
+            .also {
```

Comment:
@dsvoronin 
`HttpServer#start` blocks the main thread. You fixed this in the tests, but not here